src/uu/kill/src/kill.rs

```diff
@@ -168,32 +173,41 @@ fn print_signals() {
             print!(" ");
         }
     }
+    Ok(())
 }
 
-fn list(arg: Option<String>) {
+fn list(arg: Option<String>) -> UResult<()> {
     match arg {
-        Some(ref x) => print_signal(x),
-        None => print_signals(),
-    };
+        Some(ref x) => Ok(print_signal(x)?),
+        None => Ok(print_signals()?),
+    }
```

Comment:
```suggestion
        Some(ref x) => print_signal(x),
        None => print_signals(),
    }
```

Reply:
Or if you make `print_signals` not return `UResult`:
```suggestion
        Some(ref x) => print_signal(x),
        None => {
            print_signals();
            Ok(())
        }
    }
```